extensions/aws/processors/ListS3.h

```diff
@@ -86,7 +91,7 @@ class ListS3 : public S3Processor {
   bool write_object_tags_ = false;
   bool write_user_metadata_ = false;
   bool requester_pays_ = false;
-  std::unique_ptr<minifi::utils::ListingStateManager> state_manager_;
+  std::unique_ptr<minifi::utils::ListingStateManager> state_manager_ = nullptr;
```

Comment:
The default constructor of `unique_ptr` does the same thing, no need to explicitly write `nullptr`

Reply:
You are right, reverted. This was due to temporarily turning it into a raw pointer and it stayed that way.